test/liblll/EndToEndTest.cpp

```diff
@@ -386,6 +386,30 @@ BOOST_AUTO_TEST_CASE(keccak256_32bytes)
 		fromHex("b10e2d527612073b26eecdfd717e6a320cf44b4afac2b0732d9fcbe2b7fa0cf6")));
 }
 
+BOOST_AUTO_TEST_CASE(alloc_1)
+{
+	char const* sourceCode = R"(
+		(returnlll
+			(return (- (msize) (alloc 42))))
+	)";
+	compileAndRun(sourceCode);
+	// Note that (msize) is always a multiple of 32.
+	BOOST_CHECK(callFallback() == encodeArgs(u256(64)));
+}
+
+BOOST_AUTO_TEST_CASE(alloc_2)
+{
+	char const* sourceCode = R"(
+		(returnlll
+			(seq
+				(mstore 0x40 0xff)
+				(mstore8 (alloc 1) 42)
```

Comment:
Sure.

Actually, I think I have found an edge-case bug in alloc.  If you do (alloc 0) it will corrupt the previous word in memory.  I'll raise an issue about this later.